DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/DelegateGroups/GroupDelegatesRemove.cshtml

```diff
@@ -0,0 +1,89 @@
+  <partial name="~/Views/TrackingSystem/Shared/_NavMenuItems.cshtml" />
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    @if (errorHasOccurred) {
+      <vc:error-summary order-of-property-names="@new []{ nameof(GroupDelegatesRemoveViewModel.Confirm) }" />
+    }
+
+    <h1 class="nhsuk-heading-xl">Are you sure you would like to remove @Model.DelegateName from this group?</h1>
+  </div>
+</div>
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-one-quarter nhsuk-heading-l">
+    <div class="nhsuk-u-font-weight-bold">
+      Group name:
+    </div>
+  </div>
+  <div class="nhsuk-grid-column-three-quarters nhsuk-heading-l nhsuk-u-font-weight-normal">
+    @Model.GroupName
+  </div>
+</div>
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    <form class="nhsuk-u-margin-bottom-5" method="post" asp-action="GroupDelegatesRemove">
+      <div class="nhsuk-form-group @confirmFormErrorClass">
+        <fieldset class="nhsuk-fieldset" aria-describedby="confirm-error">
+          @if (confirmError) {
+            <span class="nhsuk-error-message" id="confirm-error">
+              <span class="nhsuk-u-visually-hidden">Error:</span> @ViewData.ModelState[nameof(GroupDelegatesRemoveViewModel.Confirm)].Errors[0].ErrorMessage
+            </span>
+          }
+          <div class="nhsuk-checkboxes">
+            <div class="nhsuk-checkboxes__item">
+              <input class="nhsuk-checkboxes__input" id="confirm" name="confirm" asp-for="Confirm" />
+              <label class="nhsuk-label nhsuk-checkboxes__label" for="confirm">
+                I am sure that I wish to remove @Model.DelegateName from this group.
+              </label>
+            </div>
+          </div>
+        </fieldset>
+      </div>
+
+      <div class="nhsuk-form-group">
+        <fieldset class="nhsuk-fieldset" aria-describedby="removeProgress-error">
+          <div class="nhsuk-checkboxes">
+            <div class="nhsuk-checkboxes__item">
+              <input class="nhsuk-checkboxes__input" id="removeProgress" name="removeProgress" asp-for="RemoveProgress" aria-describedby="removeProgress-item-hint" />
+              <label class="nhsuk-label nhsuk-checkboxes__label" for="removeProgress">
+                Remove delegate from all the related enrolments where course is not yet complete.
+              </label>
+              <div class="nhsuk-hint nhsuk-checkboxes__hint" id="removeProgress-item-hint">
+                Optionally all enrolments on courses that are incomplete and are associated with the delegate's membership of this group can also be removed.
+                This facility should not be used if the delegate may have been enrolled on the same course(s) because of membership of another group.
+              </div>
+            </div>
+          </div>
+        </fieldset>
+      </div>
+
+      <input type="hidden" asp-for="GroupName" />
+      <input type="hidden" asp-for="DelegateName" />

```

Comment:
Are these hidden input necessary for something?

Reply:
Yeah, for the invalid model state return. Otherwise they'll be blank unless we look them up again